--- opensc-0.11.3/src/tools/opensc-explorer.c.orig	2007-06-30 10:55:58.000000000 +0200
+++ opensc-0.11.3/src/tools/opensc-explorer.c	2007-08-12 00:43:16.653483641 +0200
@@ -1148,7 +1148,6 @@
 	return -1;
 }
 
-#if 0
 static int do_debug(int argc, char **argv)
 {
 	int i;
@@ -1170,7 +1169,6 @@
 	}
 	return 0;
 }
-#endif
 
 
 static int do_pksign(int argc, char **argv)
@@ -1454,6 +1452,7 @@
  { "exit",	do_quit,	"quit this program"			},
  { "update_record", do_update_record, "update record"			},
  { "update_binary", do_update_binary, "update binary"			},
+ { "debug",	do_debug,	"set the debug level"			},
  { NULL, NULL, NULL }
 };
 
