--- opensc-0.7.0/src/libopensc/card-flex.c.orig	Thu May  2 15:54:27 2002
+++ opensc-0.7.0/src/libopensc/card-flex.c	Thu Mar 13 09:59:11 2003
@@ -26,6 +26,7 @@
 static const char *flex_atrs[] = {
 	"3B:95:94:40:FF:63:01:01:02:01",       /* Cryptoflex 16k */
 	"3B:85:40:20:68:01:01:05:01",          /* Cryptoflex 8k */
+	"3B:95:15:40:FF:68:01:02:02:04",       /* Cryptoflex 8k v2 */
 	"3B:19:14:55:90:01:02:02:00:05:04:B0",
 	"3B:02:14:50",                         /* Multiflex 3K */
 	"3B:19:14:55:90:01:02:01:00:05:04:B0", /* Multiflex 4K */
