--- opensc-0.8.0/src/pkcs11/Makefile.am.orig	Fri Aug 15 10:45:23 2003
+++ opensc-0.8.0/src/pkcs11/Makefile.am	Thu Aug 21 13:13:26 2003
@@ -1,7 +1,7 @@
 # Process this file with automake to create Makefile.in
 
 bundledir = $(BUNDLEDIR)
-libdir = @libdir@/pkcs11
+pkcs11dir = @libdir@/pkcs11
 
 MAINTAINERCLEANFILES = Makefile.in
 
@@ -16,7 +16,8 @@
 	framework-pkcs15init.c debug.c
 INC = sc-pkcs11.h
 
-lib_LTLIBRARIES = opensc-pkcs11.la libpkcs11.la
+lib_LTLIBRARIES = libpkcs11.la
+pkcs11_LTLIBRARIES = opensc-pkcs11.la
 
 opensc_pkcs11_la_SOURCES = $(SRC) $(INC)
 opensc_pkcs11_la_LDFLAGS = -module -avoid-version
