--- opensc-0.10.0/src/signer/Makefile.am.orig	2005-10-31 10:30:41.000000000 +0100
+++ opensc-0.10.0/src/signer/Makefile.am	2005-11-07 14:08:45.000000000 +0100
@@ -9,7 +9,7 @@
 
 EXTRA_DIST = testprog.c
 
-INCLUDES = -I$(srcdir)/npinclude $(CPPFLAGS) $(OPENSSL_CFLAGS) $(ASSUAN_CFLAGS)
+INCLUDES = -I$(srcdir)/npinclude $(CPPFLAGS) $(OPENSSL_CFLAGS) $(LIBASSUAN_CFLAGS)
 
 if HAVE_OPENSSL
 if HAVE_ASSUAN
@@ -18,7 +18,7 @@
 endif
 
 opensc_signer_la_LDFLAGS = -module -avoid-version
-opensc_signer_la_LIBADD = @LIBOPENSC@ $(OPENSSL_LIBS) $(ASSUAN_LIBS)
+opensc_signer_la_LIBADD = @LIBOPENSC@ $(OPENSSL_LIBS) $(LIBASSUAN_LIBS)
 opensc_signer_la_CFLAGS = $(AM_CFLAGS) -DXP_UNIX
 opensc_signer_la_SOURCES = \
 	opensc-crypto.c opensc-support.c signer.c stubs.c dialog.c
--- opensc-0.10.0/configure.in.orig	2005-10-31 10:27:42.000000000 +0100
+++ opensc-0.10.0/configure.in	2005-11-07 14:08:17.000000000 +0100
@@ -294,8 +294,6 @@
 	AM_CONDITIONAL(HAVE_ASSUAN, false)
 	ASSUAN_MSG=no
 	])
-AC_SUBST(ASSUAN_CFLAGS)
-AC_SUBST(ASSUAN_LIBS)
 
 dnl enable/disable logging of sensitive apdu data
 AC_ARG_ENABLE(sensitive-logging,
